.github/ISSUE_TEMPLATE/07-feature_request.yml

```diff
@@ -10,24 +9,12 @@ body:
     id: fr-descrip
     attributes:
       label: Describe the feature request
-      placeholder: A clear and concise description of the feature you would like and why it should be added.
-    validations:
-      required: true
-  - type: input
-    id: ort-version
-    attributes:
-      label: ONNX Runtime Version
-      placeholder: ex. 1.12.1
+      placeholder: A clear and concise description of the feature you would like, why it should be added, and any alternative solutions or features you've considered. Also provide any additional context necessary for your issue to be addressed (e.g., screenshots, GPU model & memory, etc.).
     validations:
       required: true
   - type: textarea
-    id: alternatives
+    id: use-case
     attributes:
-      label: Alternatives
-      placeholder: A clear and concise description of any alternative solutions or features you've considered.
+      label: Describe scenario use case

```

Comment:
Add placeholder: "The capability and/or customer usage scenario this request supports."